src/functionalTest/resources/application.yaml

```diff
@@ -12,7 +12,7 @@ azure:
       namespace: ${PAYMENTS_QUEUE_NAMESPACE}
       write-access-key: ${PAYMENTS_QUEUE_WRITE_ACCESS_KEY}
       write-access-key-name: ${PAYMENTS_QUEUE_WRITE_ACCESS_KEY_NAME}
-      queue-name: payments
+      queue-name: ${PAYMENTS_QUEUE_NAME}
```

Comment:
you need to set value for this env var in jenkins cnp ?

Reply:
true. it's present for app only :facepalm:

Reply:
Also you will need to set env var in flux for functional tests. Worth adding that first and then merging this so both staging and non staging func tests works

Reply:
will have to include in existing chart bump